src/utils/errors.ts

```diff
@@ -1,23 +1,12 @@
 import { HttpError } from 'http-errors';
 
```

Comment:
We should amend the documentation in line 51:

> **IMPORTANT** This underlying error is **not** escaped in any way

Reply:
Good point [8e4ab04](https://github.com/auth0/nextjs-auth0/pull/864/commits/8e4ab04ca54b750c4cb1ca988855afec6cd9ffcf)